MAINTAINERS-example

```diff
@@ -1,33 +1,23 @@
-# This project, topsikrit, has top-secret awesome sauce.
-#
-# Btw, note that this comment was human-edited and will be preserved
-# even if you add/remove maintainers using the gem. Nifty!
-
-maintainers: {
-    version: 1
-    issues: "https://github.com/csprojects/topsikrit/issues"
-    # This is a very interesting comment. It will also be preserved.
-    roles: {
-        core: {
-            "ada": {},
-            "turing": {},
-        },
-        subsystem: {
-            "gracehopper": { subsystem: "ENIAC" },
-        },
+{
+  "version": 1,
+  "maintained": false,
```

Comment:
So ... this file is a bad example in this regard. The key should either be true/absent and there should be 1 or more people listed, or it should be false and there should be no people listed.